src/net_common.c

```diff
@@ -34,40 +35,52 @@
 
 #define KEEPALIVE_PERIOD 1
 
+static struct
+{
+    net_protocol_t protocol;
+    char *name;
```

Comment:
`name` is initialized with a string literal and should therefore be declared `const`

Reply:
Considered doing this but I was lazy to fix `NET_WriteString()`. Now done.